docs/src/pages/components/icons/FontAwesomeIcon.tsx

```diff
@@ -30,7 +18,13 @@ export default function FontAwesomeIcon() {
   }, []);
 
   return (
-    <div className={classes.root}>
+    <Box
+      sx={{
+        '& > *': {
```

Comment:
We can increase specificity, avoid any margin issue

```suggestion
        '& > :not(style)': {
```

x the other cases

Reply:
Resolved